src/service.js

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios'
+
+const HOST = "127.0.0.1"
+const PORT = "3001"
+
+export function createUser(user) {
+axios.post(`https://${HOST}:${PORT}/createUser`,
+                    {"user": user},
+                    {wihCredentials:true},
+            ).then(response=>{console.log("registration res: ", response)}
+            ).catch(error => {console.log("registration error: ", error)})
+}
+
+export function validate(usr) { //returns array of invalid keys
```

Comment:
Nazwalbym to validateUser, zeby bylo wiadomo co to wlasciwie waliduje + jakas dokumentacja by sie przydala.